### ISSUE TITLE
smtplib Authentication Error 454

### ISSUE THREAD

**Issue by harleylang** [emersion/hydroxide#79]
I have setup `go` and `hydroxide` on my raspberry pi. I am able to serve but am receiving this error when I attempt to authenticate in python:

```
mail.login('**********@pm.me','**********')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/smtplib.py", line 730, in login
    raise last_exception
  File "/usr/lib/python3.7/smtplib.py", line 721, in login
    initial_response_ok=initial_response_ok)
  File "/usr/lib/python3.7/smtplib.py", line 642, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (454, b'4.7.0 json: cannot unmarshal number 5368709120 into Go struct field User.User.MaxSpace of type int')
```

**Comment by harleylang** [comment]
I think I have successfully traced the error but `go` is foreign to me so I have no way to test this solution.

The output is providing an `int` of `5368709120` to `MaxSpace`

Looks like `MaxSpace` is only in `protonmail/users.go`: https://github.com/emersion/hydroxide/search?q=MaxSpace&unscoped_q=MaxSpace

As an `int` it cannot accept the number above as per: https://golang.org/ref/spec#Numeric_types

This is because my raspberry pi is 32bit and `int` therefore defaults to 32bit. https://stackoverflow.com/questions/21491488/what-is-the-difference-between-int-and-int64-in-go

**Comment by emersion** [comment]
Can you send a patch making it an `int64` instead?

**Comment by harleylang** [comment]
I retested the python script (up until the login command) and it words with the above change to `int64`

**Comment by harleylang** [comment]
Excellent thanks for accepting my pull request. I will test again on my raspberry pi later this morning to confirm that this is no longer an issue.

**Comment by harleylang** [comment]
Can confirm this is fixed with #80
